hardware_interface/test/test_component_parser.cpp

```diff
@@ -277,33 +276,31 @@ class TestComponentParser : public Test
       <state_interface name="position"/>
     </joint>
   </ros2_control>
-  <ros2_control name="2DOF_System_Robot_ForceTorqueSensor" type="sensor">
+  <ros2_control name="RRBotForceTorqueSensor2D" type="sensor">
     <hardware>
-      <plugin>ros2_control_demo_hardware/2D_Sensor_Force_Torque</plugin>
+      <plugin>ros2_control_demo_hardware/ForceTorqueSensor2DHardware</plugin>
       <param name="example_param_read_for_sec">0.43</param>
-      <param name="frame_id">kuka_tcp</param>
     </hardware>
     <sensor name="tcp_fts_sensor">
-      <state_interface name="fx">
-        <param name="lower_limits">-100</param>
-        <param name="upper_limits">100</param>
-      </state_interface>
-      <state_interface name="fy">
-        <param name="lower_limits">-100</param>
-        <param name="upper_limits">100</param>
-      </state_interface>
+      <state_interface name="fx"/>
```

Comment:
I would like to have limits here... Especially with FTS is that important since one can easily get measurements outside calibrated range.

Reply:
This is often defined per-axis. see: https://www.ati-ia.com/products/ft/ft_models.aspx?id=Mini45

Reply:
Let's discuss that on the roadmap repo if you don't mind. This PR simply brings the examples we have up to date.

Reply:
OK